magento/graphql-ce
# Resolve coupling between objects in `\Magento\QuoteGraphQl\Model\Cart\SetBillingAddressOnCart`

### Description (*)
Fixed issue magento/graphql-ce#468

### Fixed Issues (if relevant)
1.  magento/graphql-ce#468

### Manual testing scenarios (*)
1. Go to `/dev/tests/static/` (CLI `cd /dev/tests/static/` from project root folder)
2. Prepare your `phpunit.xml` from `phpunit.xml.dist`
3. run `../../../vendor/bin/phpunit`
4. Tests results should be green

![image](https://user-images.githubusercontent.com/13456702/54473531-ad2afc80-47e1-11e9-9182-4e3dbfda7faa.png)

### Contribution checklist (*)
